draft-irtf-cfrg-opaque.md

```diff
@@ -1878,6 +1895,14 @@ implementation considerations.
   of the derivation of OPRF keys via a single PRF. As long as the derivation
   of different OPRF keys from a single OPRF has different PRF inputs, the
   protocol is secure. The choice of such inputs is up to the application.
+- {{JKX18}} comments on a strong defense against essentially the only
+  inherent remaining weakness of a "strong aPAKE", namely, offline
+  dictionary attacks upon server compromise or honest-but-curious servers.
```

Comment:
excuses! i was confusing when i wrote 
> more than honest-but-curious
which i intended to mean that the server is more "evil" than honest-but-curious. i guess, a better phrase would've been to just say: maliciuos. not sure if this really makes any difference, but in my understanding there is two cases: 1/ the server is compromised by an outside attacker who has then access to all the "records" and starts bruteforcing those and 2/ the server is operated by a malicious actor who is bruteforcing records entrusted to it. semantics,  schmemantics, dunno if it is necessary to complicate things in this regard. either way the server is compromised i guess. so let's leave it at that?
apologies again for being confusing and causing more work than needed.

Reply:
No problem :) I totally understand your point. If you find a way to express this clearly please don't hesitate to open a PR!